pymeasure/instruments/edwards/NXDS.py

```diff
@@ -0,0 +1,52 @@
+#
+# This file is part of the PyMeasure package.
+#
+# Copyright (c) 2013-2020 PyMeasure Developers
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in
+# all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+# THE SOFTWARE.
+#
+
+# from pymeasure.instruments import Instrument
+
+from time import sleep
+from pymeasure.instruments import Instrument
+
+class NXDS(Instrument):
+    """ Represents the Edwards nXDS (10i) Vacuum Pump
+    and provides a low-level interaction with the instrument.
+
+    This instrument is constructed to only start and stop pump.
+    """
+
+    def __init__(self, resourceName, **kwargs):
+        super(NXDS, self).__init__(
+            resourceName,
+            "Edwards NXDS Vacuum Pump",
+            **kwargs
+        )
+	
```

Comment:
I understand the pump is non SCPI instrument with serial interface, so I would guess that `includeSCPI` parameter should be set to `False`. In general, you could have a look at the https://github.com/pymeasure/pymeasure/blob/master/pymeasure/instruments/fluke/fluke7341.py instrument to get inspiration on how to define the `__init__` function for a non SCPI serial instrument.